beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -5322,7 +5322,8 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             BeaconState::Bellatrix(_)
             | BeaconState::Capella(_)
             | BeaconState::Deneb(_)
-            | BeaconState::Electra(_) => {
+            | BeaconState::Electra(_)
+            | BeaconState::Fulu(_) => {
```

Comment:
Can we do `fork.after_bellatrix` here? the argument of using a match to force us to think about the fork is broken here, because we don't know what EIPs will exist in the fork and if we need a new handler. So now we are getting the worst of both worlds. a 1980 lines boilerplate PR and no guarantees or correctness